src/vmm/src/config.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+
+use std::convert::TryFrom;
+use std::fmt;
+use std::path::PathBuf;
+use std::result;
+
+/// Errors encountered converting the `*Config` objects.
+pub enum ConversionError {
+    /// Failed to parse the string representation for the kernel.
+    ParseKernel(String),
+    /// Failed to parse the string representation for guest memory.
+    ParseMemory(String),
+    /// Failed to parse the string representation for the vCPUs.
+    ParseVcpus(String),
+}
+
+impl fmt::Display for ConversionError {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        use ConversionError::*;
+        match self {
+            ParseKernel(ref s) => write!(f, "Invalid input for memory: {}", s),
+            ParseMemory(ref s) => write!(f, "Invalid input for memory: {}", s),
+            ParseVcpus(ref s) => write!(f, "Invalid input for memory: {}", s),
```

Comment:
These should be updated with kernel/memory/vcpus.

Reply:
Done